### ISSUE TITLE
feature request: Add sorting ability for items

### ISSUE THREAD

**Issue by Hoimar** [nxt3AT/sankeydiagram.net#41]
It looks like currently, items are sorted alphabetically by default.
It would be nice to either sort them manually (by order of appeareance in the markup).
Or alternatively, extend the syntax to be able to specify a sorting order for the items below a node.

**Comment by JonasDoesThings** [comment]
I have added a setting yesterday, that enables sorting of nodes on the same level by their respective line number.   
Previously / without the setting enabled - sorting primarily relies on alphabetical order of the nodes. 

Will go live with the next update if I don't discover any issues.

Example Input:
```
zzz [1] target a
yyy [1] target a
zsource a [10] target a
source a2 [1] target a
source a3 [15] target a
xxx [2] target a
target a [1] target b
```

## Old Behavior:
<img width="1059" alt="image" src="https://github.com/user-attachments/assets/40d3b665-2869-4c29-bbc6-745a82e1f68e">  

([Edit This Diagram](https://sankeydiagram.net/?content=F4gEG0EYF1QFwIYCcDmBTOoECgCe+IZ5l1MdgBnAewFckBjNLQgBlkVQy22rsawBMhdiS45eDJggDMhAKwjOZbAA81EAYtLcO2hMOJLQAIyA&))  
  
---

## New Behavior WITH SETTING ENABLED:
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/d17f5d29-e453-4a11-b757-6a822a91071c">

([Edit This Diagram](https://dev.sankeydiagram-net.pages.dev/?content=F4gEG0EYF1QFwIYCcDmBTOoECgCe+IZ5l1MdgBnAewFckBjNLQgBlkVQy22rsawBMhdiS45eDJggDMhAKwjOZbAA81EAYtLcO2hMOJLQAIyA&sankey-settings-sort-nodes-by-linenumber=true))

**Comment by JonasDoesThings** [comment]
Released in 1.5.1. You can enable "Sort Nodes by Line Number" in the settings to sort nodes on the same level.
